sdk/opendp/smartnoise/metadata/collection.py

```diff
@@ -214,13 +221,16 @@ def load_table(self, schema, table, t):
         row_privacy = bool(t["row_privacy"]) if "row_privacy" in t else False
         max_ids = int(t["max_ids"]) if "max_ids" in t else 1
         sample_max_ids = bool(t["sample_max_ids"]) if "sample_max_ids" in t else None
+        use_dpsu = bool(t["use_dpsu"]) if "use_dpsu" in t else None
+        clamp_counts = bool(t["clamp_counts"]) if "clamp_counts" in t else None
```

Comment:
Why is clamp_columns default a bool and clamp_counts default None?

Reply:
Same for dpsu

Reply:
The code defaults to None for those params if they aren't present, since they are optional in metadata, and we want the metadata to round-trip back to disk without adding properties that weren't set originally.

Reply:
makes sense